zerver/tornado/views.py

```diff
@@ -72,7 +72,7 @@ def get_events_backend(request: HttpRequest, user_profile: UserProfile,
                        event_types: Optional[Sequence[str]]=REQ(default=None, validator=check_list(check_string),
                                                                 intentionally_undocumented=True),
                        dont_block: bool=REQ(default=False, validator=check_bool),
-                       narrow: Iterable[Sequence[str]]=REQ(default=[], validator=check_list(None),
+                       narrow: Iterable[Sequence[str]]=REQ(default=[], validator=check_list(check_list(check_string)),
```

Comment:
I'm not sure if this is true in this code path, but in `GET /messages`, these can be a List[Dict[str, str]]`` instead.

Reply:
As far as I can tell, this one doesn’t support that format; it’s passed to `build_narrow_filter` whose annotation and code both expect the array of arrays format.